### DISCUSSION TITLE
I accidentally deleted my account

### DISCUSSION THREAD

**Discussion by vladn1994** [community/community#127933] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi everyone,

I just made a really silly mistake. In an attempt to remove a repository, I removed my GitHub account (I don't know how it happened; somehow, my brain was switched off for 5 seconds). I really need the content from there, and I am wondering if there is any fast action I could take to reverse the deletion (I was reading about some grace period). I would appreciate any advice.
Thank you!

Vlad

**Comment by ebndev** [comment]
Hi @vladn1994,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
